Exception is thrown during Project.getDescription()